discover/index.html

```diff
@@ -27,72 +27,72 @@ <h2 class="text-left"><i class="fas fa-fw fa-plane"></i> Getting Started</h2>
+  <li><i class="fas fa-fw fa-unlock"></i> <code>Basemap-Overlay</code> – Base map of vector features, highways and streets, cities, address points, parcels, etc. (transparent background)
+  <li><i class="fas fa-fw fa-unlock"></i> <code>Basemap-Terrain</code> – Base map of hillshade, highways and streets, parks, forests, water, etc.
+  <li><i class="fas fa-fw fa-unlock"></i> <code>Basemap-Topo</code> – Base map of US Geological Survey topographic maps
+  <li><i class="fas fa-fw fa-unlock"></i> <code>DOQ 1990s BW</code> – 1-meter statewide black-and-white aerial photography, collected in the 1990s
+  <li><i class="fas fa-fw fa-unlock"></i> <code>HRO 2012 RGB</code> – 12.5-cm natural-color aerial photography of the Wasatch Front, collected in 2012 
+  <li><i class="fas fa-fw fa-unlock"></i> <code>NAIP {year} RGB</code> – 1-meter statewide natural-color aerial photography
     <ul class="dotless">
-      <li><em>Available Years: 2006 2009 2011 2014 2016</em>
+      <li><em>Available Years: 2006, 2009, 2011, 2014, 2016</em>
     </ul>
   <li><i class="fas fa-fw fa-unlock"></i> <code>NAIP {year} NRG</code> – 1-meter statewide color infrared aerial photography
     <ul class="dotless">
-      <li><em>Available Years: 2006 2011 2014 2016</em>
+      <li><em>Available Years: 2006, 2011, 2014, 2016</em>
     </ul>
-  <li><i class="fas fa-fw fa-unlock"></i> <code>Hillshade</code> – 10 meter statewide combined hillshade and slope
+  <li><i class="fas fa-fw fa-unlock"></i> <code>Hillshade</code> – 10-meter statewide combined hillshade and slope
 </ul>
 <h3 class="text-left"><i class="far fa-fw fa-thumbs-up"></i> Best Practices</h3>
 <ul>
-  <li>Information about <a href="{{ "/data/googleimagery/" | prepend: site.baseurl }}">transformations between different coordinate systems</a>.
+  <li>Click <a href="{{ "/data/googleimagery/" | prepend: site.baseurl }}">here</a> for more information about transformations between different coordinate systems.
+  <li>Click <a href="{{ "/data/googleimagery/" | prepend: site.baseurl }}">here</a> for more information about using Google Imagery and Discover Services.
   <li>It is strongly suggested that you use <code>WMTS</code> for the best performance.
-  <li>More information about using the <a href="{{ "/data/googleimagery/" | prepend: site.baseurl }}">Google Imagery and Discover Services</a>.
-  <li>If you will be or already are printing base maps from the web and are experiencing blank or empty output, read this blog post about <a href="{{"/printing-web-maps-with-discover-services/" | prepend:site.baseurl }}">Printing Web Maps with Discover Services</a> for our solution.
+  <li>If you are, or will be, printing base maps from the web and are experiencing blank or empty output, read about our solution <a href="{{"/printing-web-maps-with-discover-services/" | prepend:site.baseurl }}">here</a>. 
 </ul>
 <h3 class="text-left"><i class="far fa-fw fa-chart-bar"></i> Statistics</h3>
```

Comment:
I think this whole section can be removed. It provides nothing of use to the user.